src/clj/ataru/applications/application_store.clj

```diff
@@ -178,6 +179,25 @@
        (map ->kebab-case-kw)
        (latest-versions-only)))
 
+(defn get-application-list-by-ssn
```

Comment:
We'd have to change all the other variants then as well:

* get-application-list-by-hakukohde
* get-full-application-list-by-hakukohde

etc. so that we'd be consistent